dagshub/upload/wrapper.py

```diff
@@ -24,6 +26,66 @@ def get_default_branch(src_url, owner, reponame, auth):
     return res.json().get('default_branch')
 
 
+def create_repo(repo_name, description="", private=False, auto_init=False, gitignores="", license="", readme="", template="notebook-template"):
+    import logging
+    logging.basicConfig(level=logging.DEBUG)
```

Comment:
Looks like all the logging below should be removed, so no need for a logger anyway